torch/_subclasses/meta_utils.py

```diff
@@ -327,14 +329,30 @@ def empty_create(inner_t, inner_src, symbolic_context=symbolic_context):
                     if t.requires_grad and not is_leaf:
                         with torch.enable_grad():
                             r = r.clone()
-                elif is_batchedtensor(t):
+                elif is_functorch_wrapped_tensor(t):
                     # Wraps a BatchedTensor in a FakeTensor
                     def _to_fake_tensor(t):
                         if is_batchedtensor(t):
                             ft = _to_fake_tensor(get_unwrapped(t))
                             lvl = maybe_get_level(t)
                             bdim = maybe_get_bdim(t)
                             r = _add_batch_dim(ft, bdim, lvl)
+                        elif is_gradtrackingtensor(t):
+                            r = self.meta_tensor(
+                                get_unwrapped(t),
+                                shape_env=shape_env,
+                                callback=callback,
+                                source=source,
+                                symbolic_context=symbolic_context,
+                            )
+                            assert is_gradtrackingtensor(r) is False, r
```

Comment:
I'm confused, the result of this should be a GradTrackingTensor holding a FakeTensor